src/_helpers/theme.js

```diff
@@ -0,0 +1,32 @@
+import { spacings } from '../tokens';
+import theme from '../tokens/theme';
+
+const colorRange = [100, 200, 300, 400, 500, 600, 700, 800, 900];
+
+const makePxValues = (value) => {
```

Comment:
`makePxValue` my bad it should be singular.

Reply:
Changed